### PR TITLE
[Doc] Remove tablet_writer_rpc_timeout_sec

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Signed-off-by: EsoragotoSpirit <wanglichen@starrocks.com>

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

File: docs/faq/loading/Insert_into_faq.md [hunk 1]
```diff
@@ -6,13 +6,14 @@ It is not recommended to insert data piece by piece to OLAP. It is usually inser
...
 ## 'Insert into select' task reports error: index channel has intoleralbe failure
 
-This can be solved by changing the timeout limit for stream load RPC. Also, change the following two items in fe.conf and be.conf to larger values (you can also make the adjustment in the Manager page):
+You can solve this problem by changing the timeout duration for the Stream Load RPC. Change the following item in **be.conf** and restart the machines to allow the change to take effect:
 
-```plain text
-streaming_load_rpc_max_alive_time_sec=2400
-tablet_writer_open_rpc_timeout_sec=120
-```
+`streaming_load_rpc_max_alive_time_sec`: The RPC timeout for Stream Load. Unit: Seconds. Default:
```
Comment (evelynzhaojie) [inline docs/faq/loading/Insert_into_faq.md:11]: the part after "default“ is missing

**Review by amber-create — DISMISSED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Comment by EsoragotoSpirit** [comment]
@mergify backport branch-3.2

**Pull request merged**